WalletWasabi.Fluent/Styles/Themes/Base.axaml

```diff
@@ -1,25 +1,27 @@
 <Styles xmlns="https://github.com/avaloniaui"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
   <Styles.Resources>
-    <Color x:Key="SystemAccentColorLight1">#FF82C690</Color>
-    <Color x:Key="SystemAccentColorLight2">#FF92CF9E</Color>
-    <Color x:Key="SystemAccentColorLight3">#FFA2D8AD</Color>
-    <Color x:Key="SystemAccentColorDark1">#FF5BA96C</Color>
-    <Color x:Key="SystemAccentColorDark2">#FF449556</Color>
-    <Color x:Key="SystemAccentColorDark3">#FF2E8041</Color>
-    <Color x:Key="SystemAccentColor">#FF72BD81</Color>
     <Color x:Key="SystemErrorTextColor">#D85050</Color>
+    <Color x:Key="NavBarTintColor">#134074</Color>
+    <Color x:Key="AcrylicFallbackColor">#1F1F1F</Color>
+
+    <SolidColorBrush x:Key="NavBarAccentColor">#97D234</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarAccentColorPressed">#719B2A</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarItemPressedAccentColor">#719B2A</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarItemDisabledBrush">#66FFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarItemPointerOverBrush">#FFFFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarItemPointerOverBackgroundBrush">#19FFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarItemPressedBrush">#FFFFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarItemSelectedBrush">#FFFFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarItemSelectedUnfocusedBrush">#FFFFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarItemSelectedPointerOverBrush">#FFFFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarItemSelectedPressedBrush">#FFFFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarItemForegroundBrush">#FFFFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="NavBarSeparatorBrush">#FFFFFFFF</SolidColorBrush>
```

Comment:
Just a question.
All of these are the same both on Dark and Light themes?

Reply:
Yes, nav bar doesnt change on both themes now